src/types/pysuper.rs

```diff
@@ -0,0 +1,26 @@
+use crate::ffi;
+use crate::type_object::PyTypeInfo;
+use crate::types::{PyTuple, PyType};
+use crate::{AsPyPointer, Py, PyAny, PyErr, PyResult, Python};
+
+/// Represents a Python `super` object.
+///
+/// This type is immutable.
+#[repr(transparent)]
+pub struct PySuper(PyAny);
+
+pyobject_native_type_core!(PySuper, ffi::PySuper_Type, #checkfunction=ffi::PyType_Check);
+
+impl PySuper {
+    pub fn new<'py>(py: Python<'py>, ty: &'py PyType, obj: &'py PyAny) -> PyResult<&'py PySuper> {
```

Comment:
It would be nice to point users to the `super()` docs in Python so that it becomes clear what combination of arguments are valid.

Reply:
Done! I add link to python doc and small example